### ISSUE TITLE
Incorrect parameter to authenticate function

### ISSUE THREAD

**Issue by justabuzz** [konsumer/aws-cognito-example#1]
Hi,

Thanks for this example. I believe the authentication code you use is obsolete. See Use Case 4: https://github.com/aws/amazon-cognito-identity-js.

I'm raising this issue to help others who may struggle to get this example to work after updating the SDK.

The change I did is in /src/api/user.js:

```export function login (Username, Password) {
  return new Promise((resolve, reject) => {
    var authenticationData = {
        Username : Username,
        Password : Password,
    };
    var authenticationDetails = new window.AWS.CognitoIdentityServiceProvider.AuthenticationDetails(authenticationData);
    var userData = {
        Username : Username,
        Pool : userPool
    };
    var cognitoUser = new CognitoUser(userData)
    cognitoUser.authenticateUser(authenticationDetails, {
      onSuccess: function (result) {
        console.log('access token + ' + result.getAccessToken().getJwtToken())
        resolve(result)

        // AWS.config.credentials = new AWS.CognitoIdentityCredentials({
        //     IdentityPoolId : '...' // your identity pool id here
        //     Logins : {
        //         // Change the key below according to the specific region your user pool is in.
        //         'cognito-idp.us-east-1.amazonaws.com/us-east-1_TcoKGbf7n' : result.getIdToken().getJwtToken()
        //     }
        // })
      },
      onFailure: reject
    })
  })
}```

**Comment by konsumer** [comment]
Looks good. Would you mind making a pull-request for it?

**Comment by konsumer** [comment]
resolved in #2
